client/core/core.go

```diff
@@ -2565,6 +2565,25 @@ func (c *Core) Trade(pw []byte, form *TradeForm) (*Order, error) {
 	return corder, nil
 }
 
+// Trade is used to place a market or limit order.
+func (c *Core) AccountKeys(pw []byte, host string) (*AccountKeysResponse, error) {
```

Comment:
`pw` is not used.  I was thinking a password check like in https://github.com/decred/dcrdex/pull/885/files#diff-13a4a6b12c1ae844ef90151638f194427cfdc545fceb58d52c0f6f33b81d859bR2528, but throwing out the returned crypter since it's not used to sign anything.  But perhaps AccountKeys does not need a password as long as the api methods ensure the user is authenticated to call it.  I just figure since this is a sensitive operation it should be password protected.

Reply:
It's checking the password prior to export now.